prepare-environment.nf

```diff
@@ -73,8 +73,8 @@ workflow prepare_environment {
   main:
     Channel.of("Starting environment preparation") | slack_message
     (dm_inch, wg_inch) = ( params.use_datamover ?
-            [Channel.of("$workflow.launchDir/singularity/bind/r2dt/data"), Channel.empty()]
```

Comment:
Why not just remove the wget based process? We already assume we are on the
cluster in so many other bigger ways (fetching ENA for example). Might was well
just assume it here too. It would make the code cleaner.

Reply:
Ok, I'll just remove it then. I was thinking it would be handy to have around in case we ever ran the pipeline on a cloud or something, but I guess that's kind of unlikely